org.apache.directory.Studio.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.apache.directory.Studio
+
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: /app/ApacheDirectoryStudio/ApacheDirectoryStudio
```

Comment:
like for the desktop. You should have a symlink in `/app/bin` and use the command name, no path.

Reply:
I swear I tried this and it didn't work, but now it seems to.